README.md

```diff
@@ -1,3 +1,13 @@
 # issues
 
 This repository is used for issues that are cross-repository or not bound to a specific repository.
+
+As an active project within the Sovereign Cloud Stack and and as the reference implementation for
+the infrastructure layer, we coordinate very closely there. It is very likely
+that we will first need to evaluate and align feature related issues in the context of the Sovereign
+Cloud Stack. As part of the Sovereign Cloud Stack project, various assumptions were made for the
+reference implementation which cannot be changed or extended independently as a project. If a feature
```

Comment:
... which OSISM does not want to or sometimes can not change indepdently from SCS.